rules/concepts.md

```diff
@@ -48,6 +48,151 @@ In some cases, it is useful to have a rule that is only manually triggered.
+| **Item**-based    | Those react on commands or state updates for the Item                  |
+| **Group**-based   | Those react on commands or state updates for member Items of the Group |
+| **Time**-based    | Those react at special times, e.g. at midnight, every hour, etc.       |
+| **Channel**-based | Those react on trigger channels provided by some add-ons               |
+| **Thing**-based   | Those react on thing status, e.g. change from `ONLINE` to `OFFLINE`    |
+| **System**-based  | Those react on certain system statuses, e.g. startup completed         |
+
+::: tip Triggers and Events - What's the difference?
+You might have heard or will hear about events and triggers, and wonder about the difference.
+
+An event is something that happens and is detectable by openHAB.
+The most important events for rules are time events (e.g. midnight), system events (e.g. startup completed), Item state events and Item command events.
+There are some more events, but it's not necessary to mention them here.
+
+In contrast to this, a trigger identifies one of those events and makes the rule run.
+:::
+
+Here are the details for each trigger type:
+
+### **Item**-based Triggers
+
+You can listen to commands for a specific item, on status updates or on status changes (an update might leave the status unchanged).
+You can decide whether you want to catch only a specific command/status or any.
+
+A simplistic explanation of the differences between `command` and `update` can be found in the article about [openHAB core actions](/docs/configuration/actions.html#event-bus-actions).
+
+**Item**-based triggers provide some information, e.g. the received command or the received state update, see [Available Values](#available-values).
+
+When using a `received command` trigger, the rule might trigger **before** the Item's state is updated.
+Therefore, if the rule needs to know what the command was, there is an [Available Value](#available-values).
+
+### **Group**-based Triggers
+
+As with Item based event-based triggers discussed above, you can listen for commands, status updates, or status changes on the direct (but not the indirect members of a nested subgroup) members of a given Group.
+You can also decide whether you want to catch only a specific command/status or any.
+
+The [Available Values](#available-values) are populated using the Item that caused the Group trigger to fire and that Item's name is stored in an additional value.
+
+Also, as with **Item**-based triggers, when using a `received command` trigger, the rule might trigger **before** the Item's state is updated.
+Therefore, if the rule needs to know what the command was, there is an [Available Value](#available-values).
+
+### **Time**-based Triggers
+
+Dependening on the rule language/engine, there are some predefined expressions you can use, but universally supported are only [cron expressions](https://www.quartz-scheduler.org/documentation/quartz-2.2.2/tutorials/tutorial-lesson-06.html).
```

Comment:
This isn't correct. Very recently there was added a `Time is <item>` trigger that is universally supported (or at least can be made so with additions to the helper libraries. This will trigger the rule based on the date time stored in the state of a DateTime Item.

It's currently missing in the docs but it does need to be added here.

I'd also like to see mention of some of the other time based trigger options like time of day triggers in the UI. We can cover them fully on the UI page but I don't want UI users coming here and being confused when they don't even see them mentioned.

Reply:
That new trigger is really cool, but I did not notice that it was added until you told me, thanks 👍 
As a openhab-js maintainer, this is double interesting for me, I‘ll add this to openhab-js.